test/espnet2/enh/test_espnet_model.py

```diff
@@ -222,17 +191,13 @@ def test_forward_with_beamformer_net(
     inputs = random_speech[..., :ch].float()
     ilens = torch.LongTensor([16, 12])
     speech_refs = [torch.randn(2, 16, ch).float() for spk in range(num_spk)]
-    noise_ref1 = torch.randn(2, 16, ch, dtype=torch.float)
-    dereverb_ref1 = torch.randn(2, 16, ch, dtype=torch.float)
+    # noise_ref1 = torch.randn(2, 16, ch, dtype=torch.float)
+    # dereverb_ref1 = torch.randn(2, 16, ch, dtype=torch.float)
```

Comment:
Better to preserve these lines for later support of denoising and dereverberation losses.

Reply:
Done.